app/views/categories/index.html.erb

```diff
@@ -0,0 +1,33 @@
+<h1 class="mb-5 text-center card p-2 mx-5 text-primary">Categories Page</h1>
+
+<% if @categories.empty? %>
+  <p class="text-center">You have no category yet</p>
+<% else %>
+  <% @categories.each do |category| %>
+    <%= link_to category_transacts_path(category), class: 'text-decoration-none'  do %>
+      <div class="card mx-3 p-2 d-flex flex-row align-items-center">
+          <div class="mx-2">
+            <%= image_tag(category.icon, size: '110x110', alt: category.name + ' icon')  %>
+          </div>
+          <div class="card-body mx-2 d-flex flex-column justify-content-between">
+            <h5><%= category.name %></h5>
+            <p class='mt-4'> <%= category.created_at.strftime('%d %b %Y') %> </p>
+          </div>
```

Comment:
- [ ] [OPTIONAL] Great job @Fayob, your project works just fine but you may need to add some little adjustments to the UI where you are rendering the date, total amount, and the name of the category so it fits/aligns well 👍🏽 

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
<img width="339" alt="Screenshot 2023-02-17 at 17 16 34" src="https://user-images.githubusercontent.com/58692590/219709025-4462a0cb-2344-4190-b8a2-ffee81c2ecac.png">| <img width="437" alt="Screenshot 2023-02-17 at 17 25 46" src="https://user-images.githubusercontent.com/58692590/219709246-04072583-f7f5-4170-bec1-3bb8c581839c.png">